Google-Meet-Scheduler/README.md

```diff
@@ -0,0 +1,67 @@
+# Google Meet Scheduler
+
+![](http://ForTheBadge.com/images/badges/made-with-python.svg)
+
+The script [here](script.py) allows you to schedule a [google meeting](https://meet.google.com/) with multiple guests using [python](https://www.python.org/). You just need to provide the topic, time and list of email ids of the guests for the meeting and the [script](script.py) will handle the rest of the work.
+
+## Setup
+
+#### Step 1. You'll need to download a `credentials.json` file.
+
+- Go to [https://developers.google.com/calendar/quickstart/python](https://developers.google.com/calendar/quickstart/python).
+
+- Click the `Enable the Google Calendar API` button as shown below.
+<p align="center"><img height="250px" src="images/3.png"></p>
+
+- Click Next on each upcoming tab, **you don't need to change anything**.
+- Download the `credentials.json` file as shown below.
+<p align="center"><img height="200px" src="images/4.PNG"></p>
+
+- Move this file to the folder where your `script.py` resides.
+
+#### Step 2. Modify this part of the code as per your needs.
+<p align="center"><img src="images/2.PNG"></p>
+
+#### Step 3. Install all the dependencies listed in the [`requirements.txt`](requirements.txt)
+- Just Run `pip install -r requirements.txt`
+- If facing any issue installing any package, just google it :)
+
+#### Step 4. Now if you run the script for the first time then you'll be requested to grant some permissions.
```

Comment:
None of the images here got rendered in the output. It's due to messed up paths of the image